Example/FearlessUtils/ViewController.swift

```diff
@@ -7,18 +7,46 @@
 //
 
 import UIKit
+import FearlessUtils
 
 class ViewController: UIViewController {
+    private struct Constants {
+        static let address = "Dm1RyxRu8bKvVUsQpGx5e1miNbUkzSBsThhVCWdHyAjuTGR"
+        static let radius: CGFloat = 32.0
+    }
+
+    private var iconView: KusamaIconView = KusamaIconView()
+
+    override func loadView() {
+        let view = UIView()
+        view.backgroundColor = .lightGray
+
+        iconView.backgroundColor = .clear
+        iconView.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(iconView)
+
+        iconView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        iconView.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        iconView.widthAnchor.constraint(equalToConstant: 2.0 * Constants.radius).isActive = true
+        iconView.heightAnchor.constraint(equalToConstant: 2.0 * Constants.radius).isActive = true
+
+        self.view = view
+    }
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view, typically from a nib.
-    }
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+        loadIcon()
     }
 
+    private func loadIcon() {
+        do {
+            let icon = try KusamaIconGenerator().generateFromAddress(Constants.address)
```

Comment:
Why Kusama? I think its standard for Polkadot, for Kusama everyone is just reusing it

Reply:
I mean, nothing criminal and probably we can rename it as soon as Kusama will have its own algo for icon. However it might confuse people

Reply:
Right. Thanks. Fixed.